Description: Relax dependency on aeson
 Attempt to solve tricky situation with LTS-4.0 transition. LTS-4.0 contains
 aeson-0.10, so we need to build gitit with it. This patch is imported into
 additional-patches, but it is not clear whether gitit is actually user-ready.
Author: Dmitry Bogatov <KAction@gnu.org>
Last-Update: 2015-01-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gitit.cabal
+++ b/gitit.cabal
@@ -170,7 +170,7 @@ Library
                      hoauth2 >= 0.4.2 && < 0.5,
                      http-conduit >= 2.1.4 && < 2.2,
                      http-client-tls >= 0.2.2 && < 0.3,
-                     aeson >= 0.7 && < 0.10,
+                     aeson >= 0.7 && < 0.12,
                      uuid >= 1.3 && < 1.4
   if impl(ghc >= 6.10)
     build-depends:   base >= 4, syb
